src/Data/pisaClient.ts

```diff
@@ -1,15 +1,33 @@
-import { Obj, createRestApiClient, currentLanguage, load } from 'scrivito'
+import {
+  Obj,
+  createRestApiClient,
+  currentLanguage,
+  currentUser,
+  load,
+} from 'scrivito'
 import { isHomepage } from '../Objs/Homepage/HomepageObjClass'
 
-export async function pisaUrl(): Promise<string | null> {
+async function pisaBaseUrl(): Promise<string | null> {
   if (import.meta.env.FORCE_LOCAL_STORAGE) return null
+  if (!(await load(() => currentUser()))) return null
 
+  // TODO: Read out `pisa_sales_api_url` from AMS instance configuration once available
```

Comment:
* [ ] FYI This TODO can probably only be understood by the one who wrote it. Remove?